j2k/testData/fileOrElement/switch/caseWithDefaultblock.java

```diff
@@ -0,0 +1,21 @@
+//method
+void foo() {
+	switch (status) {
+	    case "init":
+	    case "dial":
+	    case "transmit":
+		return Color.BLACK;
+
+	    case "ok":
+		return 0xFF006600;
+
+	    case "cancel":
+		return 0xFF666666;
+
+	    case "fail":
+	    case "busy":
+	    case "error":
```

Comment:
Please add a test for the case when 'default' is in the middle of the list of other cases (`case "error": default: case "busy:") and make sure that this test passes.